src/open_company/resources/company.clj

```diff
@@ -122,14 +114,22 @@
 
   Ensures the slug is valid, or returns `:invalid-slug`.
 
-  If everything is OK with the proposed new company, `true` is returned.
-
-  TODO: Use prismatic schema to validate company properties."
+  If everything is OK with the proposed new company, `true` is returned."
```

Comment:
Seems this needs updated to use the schema to validate so schema validation applies to the Clojure API, not just the REST API.